mreg/models/host.py

```diff
@@ -21,6 +31,29 @@ class Meta:
     def __str__(self):
         return str(self.name)
 
+    def community(self):
+        return self.network_community
+    
+    def set_community(self, community: Community) -> bool:
+        """Set the community for this host.
+        
+        :param community: The community to set.
+        :return: True if the community was set, False otherwise
+        """
+        # We need to check that the community is applicable to the same
+        # network as one of the IP addresses of the host.
+        for ipaddress in self.ipaddresses.all(): # type: ignore
+            from mreg.models.network import Network
```

Comment:
Can probably move this import to the top of the method.

Reply:
Done!